1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+- A cluster with only one master node is usable for development, but is not highly available and may not be able to recover from failure.
 
-The abbreviated form of the Datacenter Operating System.
+#### <a name="agent-node"></a>Agent Node
 
-# <a name="dcoscluster"></a>DC/OS Cluster
+A DC/OS agent node is a virtual or physical machine on which Mesos tasks are run.
 
-A group of Mesos master and agent nodes.
+- Each agent node contains (among other things) a Mesos agent process.
+- Agent nodes may be [private](#private-agent-node) or [public](#public-agent-node), depending on agent and network configuration.
+- For more information, see [Network Security](/docs/1.8/administration/securing-your-cluster/) and [Adding Agent Nodes](/docs/1.8/administration/installing/custom/add-a-node/).
 
-# <a name="dcosmarathon"></a>DC/OS Marathon
+##### <a name="private-agent-node"></a>Private Agent Node
 
-The native Marathon instance that is the “init system” for DC/OS. It starts and monitors DC/OS applications and services.
+A private agent node is an agent node that is on a network that** ***does not* have ingress access from outside of the cluster via the cluster’s infrastructure networking.
 
-# <a name="dcosservice"></a>DC/OS service
+- The Mesos agent on each private agent node is (by default) configured with none of its resources allocated to any specific Mesos roles (`*`).
+- Most service packages install by default on private agent nodes.
+- Clusters are usually made up of mostly private agent nodes.
 
-DC/OS services are applications that are packaged and available from the public [GitHub package repositories][5]. Available DC/OS services include Mesos frameworks and other applications.
+##### <a name="public-agent-node"></a>Public Agent Node
 
-# <a name="executor"></a>Executor
+A public agent node is an agent node that is on a network that *does* have ingress access from outside of the cluster via the cluster’s infrastructure networking.
 
-A framework running on top of Mesos consists of two components: a scheduler that registers with the master to be offered resources, and an executor process that is launched on agent nodes to run the framework’s tasks. For more information about framework schedulers and executors, see the [App/Framework development guide][6].
+- The Mesos agent on each public agent node is configured with the "public_ip:true" attribute and all of its resources allocated to the `slave_public` role (`agent_public` in the future).
+- Public agent nodes are used primarily for externally facing reverse proxy load balancers, like Marathon-LB.
+- Clusters usually only have a few public agent nodes, because a single load balancer can handle proxying multiple services.
+- For more information, see [Converting Agent Node Types](/docs/1.8/administration/installing/custom/convert-agent-type/).
```

Comment:
I think so, since it applies to the whole list of items, and doesn't really stand by itself, like the other bulleted items.